portal-api/api-tests/v1/gef/files.api-test.js

```diff
@@ -61,9 +64,37 @@ describe(baseUrl, () => {
   });
 
   describe(`POST ${baseUrl}`, () => {
-    it('rejects requests that do not present a valid Authorization token', async () => {
-      const { status } = await as().post({}).to(baseUrl);
-      expect(status).toEqual(401);
+    withClientAuthenticationTests({
+      makeRequestWithoutAuthHeader: () => postMultipartForm({
+        url: baseUrl,
+        data: { parentId: mockDeal.body._id },
+        files: [],
+      }),
+      makeRequestWithAuthHeader: (authHeader) => postMultipartForm({
+        url: baseUrl,
+        data: { parentId: mockDeal.body._id },
+        files: [],
+        headers: { Authorization: authHeader }
+      }),
+    });
+
+    withRoleAuthorisationTests({
+      allowedRoles: [MAKER],
+      rolesToSkipTestsFor: [ADMIN],
+      getUserWithRole: (role) => testUsers().withBankName(testBankName).withRole(role).one(),
+      getUserWithoutAnyRoles: () => testUsers().withBankName(testBankName).withoutAnyRoles().one(),
+      makeRequestAsUser: (user) => as(user).postMultipartForm({ parentId: mockDeal.body._id }, validFiles).to(baseUrl),
+      successStatusCode: 201,
+    });
+
+    // TODO DTFS2-6626: admins cannot upload gef files - is that okay?
```

Comment:
Flagging question

Reply:
This is resolved now as we know it is deliberate because 'admins' don't have full access to write operations